### ISSUE TITLE
Create a CONTRIBUTING.md file for guidelines on contributions

### ISSUE THREAD

**Issue by anchalchaturvedi08** [kvcops/KV-Nexus#45]
Objective:
To create a CONTRIBUTING.md file that provides clear and helpful guidelines for contributing to this project. This file will outline the process contributors should follow when creating pull requests, reporting issues, or suggesting improvements.

Requirements:  
1. Outline the steps for contributing, including:
   - Cloning the repository.
   - Setting up the environment if necessary.
   - Creating a new branch and making changes.
2. Include details about code style, commit message conventions, and testing requirements (if any).
3. Mention the process for reporting issues and feature requests.
4. Add guidelines for respectful and constructive communication.

Deliverable: 
A `CONTRIBUTING.md` file that is comprehensive, clear, and structured, ensuring a smooth experience for new contributors.

**Comment by techy4shri** [comment]
What code style, commit message conventions, and testing requirements did you observe in this repo?

**Comment by anchalchaturvedi08** [comment]
In reviewing the repository, I did not find specific files that define a code style (such as `.eslintrc` or `.prettierrc`) or a documented commit message convention. There are also no dedicated testing files or a `tests` folder that would typically indicate testing requirements. I reviewed the `package.json` and `requirements.txt` files, but they did not contain any testing scripts or configurations related to code style. It seems that this project may not enforce strict guidelines for code style, commit messages, or testing.
